projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -319,14 +320,32 @@ const WeatherHeader = () => {
     if (!isWeatherShown) {
         return <View style={styles.emptyWeatherSpace} />
     }
+
+    return <WeatherController />
+}
+
+const WeatherController = React.memo(() => {
+    const query = useQuery<QueryData>(QUERY)
```

Comment:
Why not co-locate this component with the query, like it was at first?

Reply:
Yeah I wanted to do this but then the styles need to go with for the container (but they feel like the belong to the issue screen logically. What do you think?

Reply:
Ah I see! Yeah whichever makes the most sense then ☺️

Reply:
I could move the styles in - I think it makes sense to co-locate like you said. One sec will send an update.